src/main/frontend/handler/editor.cljs

```diff
@@ -3004,19 +3004,26 @@
   (cut-blocks-and-clear-selections! false))
 
 ;; credits to @pengx17
-(defn- copy-current-block-ref
-  []
+(defn copy-current-block-ref
```

Comment:
Would be good to keep the fn private since this PR hasn't changed its usage in other namespaces

Reply:
I had made it public during testing to try call it directly. Good catch. Looks like I missed that.